### DISCUSSION TITLE
CVE-2023-4863 Webp

### DISCUSSION THREAD

**Discussion by addison74** [OpenMage/magento-lts#3562] [General]
Important notice to those who requested this image format as default in OpenMage.

**Comment by fballiano** [comment]
wow, luckily it's already been fixed. I always prefer for customers to upload jpg/png (depending) and then the optimizations (resolution + compression) gets done on the fly by the CDN, to me it is the best way.

**Reply by addison74** [reply]
In addition, those who upload images in OpenMage with this format should update their browsers as soon as possible to the latest versions.
